opentreemap/importer/models/species.py

```diff
@@ -263,6 +263,7 @@ def validate_itree_code_field(self):
         else:
             self.cleaned[fields.species.ITREE_PAIRS] = pairs
 
+    @transaction.atomic
```

Comment:
Or maybe the transaction is for a different reason? Seems OK to have those two `save`s without a transaction.